lib/log.js

```diff
@@ -133,14 +133,9 @@ class Log extends EventEmitter {
 
     return Promise
             .try(() => {
-              let error = undefined;
-      
-              if (!_.isEmpty(err)) {
-                let { name, message, stack } = err;
-                error = { name, message, stack };
-              }
-      
-              let patch = _.omitBy({ error , output }, _.isEmpty);
+              let error = _.isEmpty(err) ? undefined : err
```

Comment:
this might be problematic... how does this work if the object is of type Error? how does it get serialized in redis? the idea in original code was to create a vanilla json object with the properties we want. now we are passing through an object that can serialize however _it_ wants.

thoughts?